### ISSUE TITLE
Allow to set default NODE_PATH in extension for all NodeTasks

### ISSUE THREAD

**Issue by lamba92** [node-gradle/gradle-node-plugin#150]
Use case: when building a Kotlin/JS project, the Gradle plugin will create in rootProject `build/js` a yarn environment with all required modules in all subprojects (which is very handy). When running `NodeTask`s. It would be reasonable to set once `NODE_PATH` to said directory in `NodeExtension` and always rely on the Kotlin/JS dependency resolution.

**Comment by bsautel** [comment]
I am not sure I understand correctly how the Kotlin/JS plugin works. It downloads all the npm dependencies of all the subprojects thanks to yarn in the `$rootProject/build/js`, right? Does it also generate a `package.json` file? Which Node.js interpreter does it use?

**Comment by lamba92** [comment]
Exactly. A uber yarn workspace is created there with all the dependencies of all subprojects of the build and you can use that environment to do whatever you want. 

In my [use case](https://github.com/lamba92/kotlingram/blob/master/examples/js-bot/build.gradle.kts#L117-L128) I need to webpack into a single file my application for NodeJS. I thought that `NodeExtension.nodeProjectDir` would be used as default for all `NodeTask`s but it isn't!

If you want to see for yourself how the build process works for Kotlin/JS + Webpack, try running `:examples:js-bot:productionWebpackExecutable` or `:examples:js-bot:developmentWebpackExecutable` of [kotlingram](https://github.com/lamba92/kotlingram).
